refs #190 design updates

emullaney opened the pull request:
- kerning aka letter-spacing
- line-height 
- header sizes
- bolding on highlights
  more readable site overall

refs https://github.com/rapidpro/rapidpro-community-portal/issues/190

Commit:
refs #190 design updates, + kerning aka letter-spacing,+ line-height, - header sizes, - bolding on highlights... more readable site overall

Commit:
fixes #190 -- letter-spacing on body copy

Commit:
fixes #190 smaller nav (menu), slight increase to line height.

Commit:
refs #190 design changes

Commit:
refs #190 list item font size

Commit:
refs #190 design updates to font family and list item spacing

kmtracey commented on rapidpro_community_portal/templates/base.html:
```diff
@@ -26,7 +26,7 @@
         <link href="{% static 'css/main.css' %}" type="text/css" rel="stylesheet">
         <link href="{% static 'css/flickity.css' %}" type="text/css" rel="stylesheet">
-        <link href="//fonts.googleapis.com/css?family=Open+Sans:400,700,400italic,700italic" type="text/css" rel="stylesheet">
+        <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800,300italic,400italic,600italic,700italic,800italic" type="text/css" rel="stylesheet">
```
This should stay //, not http://

emullaney replied:
Done.

Commit:
refs #190 reset http:// to // for font include on base template

kmtracey committed:
Adjust default font weight to 400 (normal)...

needed now that we have made lighter font weight available, since
otherwise we start using the extra-light font weight most everywhere.

kmtracey commented:
:+1:
